src/core/css/SearchResult.scss

```diff
@@ -73,11 +81,4 @@
 .SearchResult-actions {
   align-items: center;
   display: flex;
-}
-
-.SearchResult-icon {
-  display: inline-block;
-  float: right;
-  height: 38px;
-  width: 38px;
-}
+}
```

Comment:
Heh, not that nitpicky apparently: https://travis-ci.org/mozilla/addons-frontend/jobs/197795481#L340

You can run `npm test` before you push your PR to get test results; it will include style linting like this so you don't have to make as many changes after pushing 😄